terraform/docker/modules/pmm_server/percona-pmm.tf

```
@@ -58,6 +58,23 @@ resource "docker_image" "pmm" {
   keep_locally = true
 }
 
+locals {
+  # Extract version tag, e.g., "2.38.1" from "percona/pmm-server:2.38.1"
+  image_tag = split(":", var.pmm_server_image)[1]
```

```suggestion
  image_parts = split(":", var.pmm_server_image)
  image_tag = length(local.image_parts) > 1 ? local.image_parts[length(local.image_parts) - 1] : "latest"
```
This would prevent errors in case the user specifies an image without a tag